README.md

```diff
@@ -13,7 +13,9 @@ This repo contains samples that demonstrate configuring PreEmptive Protection JS
 - [Electron](Electron): demonstrates how you can use the JSDefender Webpack plugin to protect Electron projects.
 - [Angular](Angular): demonstrates how you can use the JSDefender Webpack plugin to protect Angular projects.
 - [Ionic](Ionic): demonstrates how you can use the JSDefender Webpack plugin to protect different Ionic projects.
+- [CLI/Web](CLI/Web): demonstrates how you can use the JSDefender Webpack plugin to protect Vanilla JavaScript projects.
+- [ReactNative/ReacNative0.64](ReactNative/ReacNative0.64): demonstrates how you can use the JSDefender Webpack plugin to protect React Native projects.
```

Comment:
@NickyArias use this instead (feel free to fix grammatical or wording issues):
[ReactNative](ReactNative): demonstrates how you can use the JSDefender Metro plugin to protect React Native projects.

Reply:
@dominikfoldi changes were applied.

Reply:
@NickyArias you forgot to update the beginning: `[ReactNative](ReactNative)`